### ISSUE TITLE
[Bug]: missing aria-label on mobile navbar opener and closer

### ISSUE THREAD

**Issue by nac62116** [mint-vernetzt/community-platform#2079]
### What happened?

A bug happened!

### Relevant log output

```shell

```
